PSReadLine/Position.cs

```diff
@@ -93,6 +93,17 @@ private static int GetEndOfLogicalLinePos(int current)
             return newCurrent;
         }
 
+        /// <summary>
+        /// Returns the position of the end of the logical line
+        /// for the 0-based specified line number.
+        /// </summary>
+        private static int GetEndOfNthLogicalLinePos(int lineIndex)
+        {
+            return GetEndOfLogicalLinePos(
+                GetBeginningOfNthLinePos(lineIndex)
+                );

```

Comment:
nitpick on style: can you please move the trailing `);` to the end of previous line?